zebra-chain/src/orchard/keys.rs

```diff
@@ -298,7 +298,7 @@ impl Eq for SpendValidatingKey {}
 
 impl From<[u8; 32]> for SpendValidatingKey {
     fn from(bytes: [u8; 32]) -> Self {
-        Self(redpallas::VerificationKey::try_from(bytes).unwrap())
+        Self(redpallas::VerificationKey::try_from(bytes).expect("bytes are valid"))
```

Comment:
How do we know that the supplied data is always valid?

This might need to be a `TryFrom` impl instead, please check with the original author of this code.
(I think Deirdre is busy with FROST right now, so we might need to open a ticket and schedule this fix later.)

Reply:
There is no difference between `unwrap()` or `expect()`, both will panic if bytes are not valid. The idea of this lint is just to use a message instead of just panicking but i don't think we should check every unwrap to see if it should panic or not while we are here.